test/modules/post/test/file.rb

```diff
@@ -32,8 +32,8 @@ def initialize(info={})
   #
   def setup
     @old_pwd = pwd
-    tmp = (directory?("/tmp")) ? "/tmp" : "%TMP%"
-    vprint_status("Setup: changing working directory to #{tmp}")
+    tmp = (directory?("/tmp")) ? "/tmp" : "%TEMP%"
+    puts "Setup: changing working directory to #{tmp}"
```

Comment:
you should be able to use vprint_status and then just set or setg VERBOSE true, then run the module

> On Jan 28, 2015, at 3:21 PM, Brent Cook <notifications@github.com> wrote:
> 
> In test/modules/post/test/file.rb <https://github.com/rapid7/metasploit-framework/pull/4654#discussion_r23724785>:
> 
> > @@ -32,8 +32,8 @@ def initialize(info={})
> >    #
> >    def setup
> >      @old_pwd = pwd
> > -    tmp = (directory?("/tmp")) ? "/tmp" : "%TMP%"
> > -    vprint_status("Setup: changing working directory to #{tmp}")
> > +    tmp = (directory?("/tmp")) ? "/tmp" : "%TEMP%"
> > +    puts "Setup: changing working directory to #{tmp}"
> That was unintentional to change to puts. I just didn't see vprint_status showing up anywhere, so had meant to switch it temporarily while debugging. What is the proper way to get vprint_status to show up in a place I can monitor?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/rapid7/metasploit-framework/pull/4654/files#r23724785>.
>